airflow/jobs/scheduler_job.py

```diff
@@ -1140,9 +1140,9 @@ def _send_sla_callbacks_to_processor(self, dag: DAG):
     def _emit_pool_metrics(self, session: Session = None) -> None:
         pools = models.Pool.slots_stats(session=session)
         for pool_name, slot_stats in pools.items():
-            Stats.gauge(f'pool.open_slots.{pool_name}', slot_stats[PoolSlotState.OPEN.value])
-            Stats.gauge(f'pool.queued_slots.{pool_name}', slot_stats[PoolSlotState.QUEUED.value])
-            Stats.gauge(f'pool.running_slots.{pool_name}', slot_stats[PoolSlotState.RUNNING.value])
+            Stats.gauge(f'pool.open_slots.{pool_name}', slot_stats["open"])
```

Comment:
Mypy complained about this as `slot_stats` is a TypedDict, so I removed this and deleted the PoolSlotState, as this is the _only_ place this class is used.

Reply:
https://github.com/python/mypy/issues/8722